onnxruntime/python/tools/quantization/onnx_model.py

```diff
@@ -369,3 +368,93 @@ def topological_sort(self):
         assert end == len(self.graph().node), "Graph is not a DAG"
         self.graph().ClearField("node")
         self.graph().node.extend(sorted_nodes)
+
+    def clean_initializers(self):
+        return ONNXModel.clean_initializers_helper(self.graph(), self.model)
+
+    @staticmethod
+    def clean_initializers_helper(graph, model):
+        """
+        Clean unused initializers including which is caused by quantizing the model.
+            return cleaned graph, and list of tensor names from this graph and all its subgraphes
+            that can not be found in this graph and its subgraphes
+        """
+        requesting_tensor_names = {}
+        requesting_tensor_names.update(
+            {input_name: 1 for node in graph.node for input_name in node.input if input_name}
+        )
+        requesting_tensor_names.update({g_out.name: 1 for g_out in graph.output if g_out.name})
+
+        new_nodes = []
+        for node in graph.node:
+            node_2_add = node
```

Comment:
naming: Is there a name for `node_2_add` that conveys what it is better to readers?